pkg/report/report_test.go

```diff
@@ -247,13 +237,13 @@ func checkReport(t *testing.T, reporter *Reporter, rep *Report, test *ParseTest)
 }
 
 func updateReportTest(t *testing.T, test *ParseTest, title string, altTitles []string, corrupted, suppressed bool,
-	typ Type, frame string) {
+	typ crash.Type, frame string) {
 	buf := new(bytes.Buffer)
 	fmt.Fprintf(buf, "TITLE: %v\n", title)
 	for _, t := range altTitles {
 		fmt.Fprintf(buf, "ALT: %v\n", t)
 	}
-	if typ != Unknown {
+	if typ != crash.UnknownType {
```

Comment:
Unknown type should not appear in any reports at all (makes sense to assert somewhere).
And here we now want to check for Unspecified type.

Reply:
Should reduce the diff as well, github UI can barely handle this PR.